index.bs

```diff
@@ -1058,7 +1060,42 @@ Creates an {{MLTensor}} associated with this {{MLContext}}.
     1. Let |promise| be [=a new promise=] in |realm|.
     1. Enqueue the following steps to [=this=].{{MLContext/[[timeline]]}}:
         1. Run these steps, but [=/abort when=] [=this=] [=MLContext/is lost=]:
-            1. Create |tensor|.{{MLTensor/[[data]]}} given |descriptor| and initialize all bytes to zeros.
+            1. Create |tensor|.{{MLTensor/[[data]]}} given |descriptor|.
```

Comment:
I suppose "initialize all bytes to zeros" should be kept for `CreateTensor()`.

Reply:
Fixed